### PR TITLE
Make entrypoint for multiple AI platform

### PR DESCRIPTION
**Pull request by sushilkm**

### CODE REVIEW

**Review by sjyang18 — APPROVED** [review]
tested and verified as expected.  Rebase once more before merge. Thank you

File: deploy/README.md [hunk 1]
```diff
@@ -387,6 +387,24 @@ NO_DELETE_PIPELINE_RESOURCE_GROUP=true
...
+
+`AI_MODEL_INFRA_TYPE` currently supports 2 values, with default being `batch-account`: 
+- `batch-account` 
+- `aks`
+
+Those users requiring to execute AI-Model against `batch-account` will not need any changes to their commands, however, those who want to execute against `aks`, can simply execute by setting the environment variable `AI_MODEL_INFRA_TYPE`.
+
+```bash
+AI_MODEL_INFRA_TYPE=aks ./deploy/setup.sh <environmentCode> <location> <pipelineName> <envTag>
+```
```
[the remark this answers is not in the document]
Comment (sushilkm) [reply deploy/README.md:406]: this change will be part of next release only.

**Review by sjyang18 — APPROVED** [review]
verified rebase issue resolved. Looks good to me.

**Review by senthilkungumaraj — APPROVED** [review]

**Comment by senthilkungumaraj** [comment]
Added an "on-hold" label. We can discuss further on the Monday weekly meeting prior to merge to main.

File: deploy/package.sh [hunk 2]
```diff
@@ -5,80 +5,103 @@
...
 
-DEPLOY_PGSQL=${13:-${DEPLOY_PGSQL:-"true"}}
+SYNAPSE_WORKSPACE_RG=${10:-${SYNAPSE_WORKSPACE_RG:-"${ENV_CODE}-pipeline-rg"}}
+SYNAPSE_WORKSPACE_NAME=${11:-$SYNAPSE_WORKSPACE_NAME}
+SYNAPSE_STORAGE_ACCOUNT_NAME=${12:-$SYNAPSE_STORAGE_ACCOUNT_NAME}
+SYNAPSE_POOL=${13:-$SYNAPSE_POOL}
+
+DEPLOY_PGSQL=${14:-${DEPLOY_PGSQL:-"true"}}
 
 MODE="batch-account$([[ $DEPLOY_PGSQL = "false" ]] && echo ",no-postgres" || echo '')"
```
Comment (sjyang18) [inline deploy/package.sh:29]: Now we can use $AI_MODEL_INFRA_TYPE variable in this PR. In the previous, I asked Raj drop this dependency, so this need updating as follows:

MODE="${$AI_MODEL_INFRA_TYPE}$([[ $DEPLOY_PGSQL = "false" ]] && echo ",no-postgres" || echo '')"

Comment (sushilkm) [reply deploy/package.sh]: updated

**Review by sjyang18 — APPROVED** [review]

**Pull request merged**